src/app/backend/iam/db/db.go

```diff
@@ -255,7 +255,7 @@ func DeleteAllUser() int64 {
 	defer db.Close()
 
 	// create the delete sql query
-	sqlStatement := `DELETE * FROM userdetails`
+	sqlStatement := `DELETE FROM userdetails`
```

Comment:
is this changing correct? what is deleted?

Reply:
Yes, with * it gives error,Thanks